add note about running make build first (#7560)
pulumi/docs: README.md

* `make ensure` resolves and installs dependencies, including the Hugo module containing website layouts and content
* `make lint` checks all Markdown files for correctness
* `make build` generates the website and writes it to `./public`

> * `make serve-static` runs a local HTTP server that serves the contents of `./public` (must run `make build` first before this will work)

Reviewer: ```suggestion
* `make serve-static` runs a local HTTP server that serves the contents of `./public` (assumes you've run `make build` first)
```
